selectinf/randomized/tests/test_selective_MLE_high.py

```diff
@@ -86,24 +86,23 @@ def test_full_targets(n=200,
                                         cov_target_score)[0]
             pval = result['pvalue']
             estimate = result['MLE']
-            intervals = np.asarray(result[['lower_confidence',
-                                           'upper_confidence']])
-
+            intervals = np.asarray(result[['lower_confidence', 'upper_confidence']])
             print("estimate, intervals", estimate, intervals)
 
             coverage = (beta[nonzero] > intervals[:, 0]) * (beta[nonzero] < intervals[:, 1])
             return pval[beta[nonzero] == 0], pval[beta[nonzero] != 0], coverage, intervals
 
-def test_selected_targets(n=2000, 
```

Comment:
A bunch of tests got deleted here -- any reason we don't test logistic and Cox still?